### ISSUE TITLE
coc-highlight error: Connection to server got closed

### ISSUE THREAD

**Issue by rollschild** [neoclide/coc.nvim#768]
Today after I updated my Vundle plugins via `:PluginUpdate`, I got the following errors when I went back to Vim: 

`[coc.nvim] Error output from highlight: Connection to server got closed. Server will not be restarted.`

I opened up `:CocCommand workspace.showOutput` and saw the following output for `coc-highlight`:
```
Error: Cannot find module 'coc.nvim'
      at Function.Module._resolveFilename (module.js:547:15)
      at Function.Module._load (module.js:474:25)
      at Module.require (module.js:596:17)
      at require (internal/module.js:11:18)
      at Object.<anonymous> (/Users/gs069673/.config/coc/extensions/node_modules/coc-highlight/lib/server/index.js:7:20)
      at Module._compile (module.js:652:30)
      at Object.Module._extensions..js (module.js:663:10)
      at Module.load (module.js:565:32)
      at tryModuleLoad (module.js:505:12)
      at Function.Module._load (module.js:497:3)
[Info  - 11:07:43] Connection to server got closed. Server will restart.
module.js:549
         throw err;
```
I checked my `coc.nvim` and it is indeed installed and compiled. Not sure why this error occurred. I installed a bunch of other coc extensions and they had no errors like this.

`node`: `8.11.4`
`neovim`: `0.3.5`

Please help. Thank you.

**Comment by iamcco** [comment]
same issue

**Comment by chemzqm** [comment]
Fixed on 1.1.2
